rcode-reader/src/main/java/eu/rssw/pct/elements/ClassElement.java

```diff
@@ -0,0 +1,19 @@
+package eu.rssw.pct.elements;
+
+public abstract class ClassElement {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this abstract class name to match the regular expression '^Abstract[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS00118)